2022/02/pan93412/testdata/td_2.txt

```diff
@@ -0,0 +1,2500 @@
+A Z
```

Comment:
Probably the same?

https://github.com/rust-tw/advent-of-code/pull/4#discussion_r1037848549

Reply:
Since that is a testdata for checking if my code works in every edge cases, I think that's ok to preserve it – users may know this is neither something to fill their input, nor something that can refer to directly. 

Besides, as the entropy of this file is relatively low, I think it doesn't take too many spaces in this repo.

Reply:
Re entropy. It is indeed highly compressible in terms of Git objects and packfiles. However, people eventually need to checkout, and it ends up occupying somewhere.

Anyhow, it is relatively small and makes this great answer complete. Going to merge it as-is. Thank you!